test/integration/integration_test.cc

```diff
@@ -1,698 +1,108 @@
-  FakeRawConnectionPtr fake_upstream_connection;
-  executeActions({[&]() -> void {
-    codec_client = makeHttpConnection(IntegrationTest::HTTP_PORT, Http::CodecClient::Type::HTTP1);
-  },
-                  [&]() -> void {
-                    codec_client->startRequest(Http::TestHeaderMapImpl{{":method", "GET"},
-                                                                       {":path", "/test/long/url"},
-                                                                       {":authority", "host"}},
-                                               *response);
-                  },
-                  [&]() -> void {
-                    fake_upstream_connection = fake_upstreams_[0]->waitForRawConnection();
-                  },
-                  // TODO: Waiting for exact amount of data is a hack. This needs to be fixed.
-                  [&]() -> void { fake_upstream_connection->waitForData(187); },
-                  [&]() -> void { fake_upstream_connection->write("bad protocol data!"); },
-                  [&]() -> void { fake_upstream_connection->waitForDisconnect(); },
-                  [&]() -> void { codec_client->waitForDisconnect(); }});
-
-  EXPECT_TRUE(response->complete());
-  EXPECT_STREQ("503", response->headers().Status()->value().c_str());
-}
-
-TEST_F(IntegrationTest, TcpProxyUpstreamDisconnect) {
-  IntegrationTcpClientPtr tcp_client;
-  FakeRawConnectionPtr fake_upstream_connection;
-  executeActions(
-      {[&]() -> void { tcp_client = makeTcpConnection(IntegrationTest::TCP_PROXY_PORT); },
-       [&]() -> void { tcp_client->write("hello"); },
-       [&]() -> void { fake_upstream_connection = fake_upstreams_[0]->waitForRawConnection(); },
-       [&]() -> void { fake_upstream_connection->waitForData(5); },
-       [&]() -> void { fake_upstream_connection->write("world"); },
-       [&]() -> void { fake_upstream_connection->close(); },
-       [&]() -> void { fake_upstream_connection->waitForDisconnect(); },
-       [&]() -> void { tcp_client->waitForDisconnect(); }});
-
-  EXPECT_EQ("world", tcp_client->data());
-}
-
-TEST_F(IntegrationTest, TcpProxyDownstreamDisconnect) {
-  IntegrationTcpClientPtr tcp_client;
-  FakeRawConnectionPtr fake_upstream_connection;
-  executeActions(
-      {[&]() -> void { tcp_client = makeTcpConnection(IntegrationTest::TCP_PROXY_PORT); },
-       [&]() -> void { tcp_client->write("hello"); },
-       [&]() -> void { fake_upstream_connection = fake_upstreams_[0]->waitForRawConnection(); },
-       [&]() -> void { fake_upstream_connection->waitForData(5); },
-       [&]() -> void { fake_upstream_connection->write("world"); },
-       [&]() -> void { tcp_client->waitForData("world"); },
-       [&]() -> void { tcp_client->write("hello"); }, [&]() -> void { tcp_client->close(); },
-       [&]() -> void { fake_upstream_connection->waitForData(10); },
-       [&]() -> void { fake_upstream_connection->waitForDisconnect(); }});
-}
+== network/connection_impl.cc ==
```

Comment:
I just saw this. I have no idea what happened because I ran the CI before pushing but apparently totally scrambled the file between step 1 and 2. Sigh.